android/src/org/ppsspp/ppsspp/CameraHelper.java

```diff
@@ -166,16 +187,41 @@ void startCamera() {
 			mCamera = Camera.open(cameraId);
 			Camera.Parameters param = mCamera.getParameters();
 
+			List<Integer> supportedPreviewFormats = param.getSupportedPreviewFormats();
+			for (int i = 0; i < supportedPreviewFormats.size(); i++) {
+				Log.i(TAG, "Supported preview format: " + i);
```

Comment:
i or supportedPreviewFormats.get(i) ?

Reply:
I'll just get rid of the logging, it was kinda pointless.